src/ui/Base/ComponentOptions.ts

```diff
@@ -616,6 +628,22 @@ export class ComponentOptions {
     }>(ComponentOptionsType.OBJECT, loadOption, optionArgs);
   }
 
+  /**
```

Comment:
Suggestion:

Builds a field condition option.

A field condition option defines a field-based condition that must be dynamically evaluated against, and satisfied by a query result item in order to initialize a result template component.

**Markup Examples:**

```
data-condition-field-author="Alice Smith, Bob Jones"
data-condition-field-not-filetype="pdf"
```